roles/secure-server/tasks/main.yml

```diff
@@ -74,24 +74,10 @@
               state=present
   notify: Restart ssh
 
-- name: Disallow password authentication
```

Comment:
Undo these changes. I want to disallow password authentication and root logins into the server. After these playbooks run, the only way into the server (except via the console, which many VPS providers can provide) is to ssh using the 'deploy' user, from the control host.

You can add other ssh keys manually to `~deploy/.ssh/authorized_keys` in order to login from other accounts that you have.

Reply:
got it.... forgot i did that.....